### ISSUE TITLE
[Enhancement] Form Field - color contrast

### ISSUE THREAD

**Issue by henrikvoetmand** [kirbydesign/designsystem#3826]
- [ ] I have written a [good issue](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Issue) 

## Describe the enhancement

**NEW TASK DESCRIPTION IN COMMENTS**

- Placeholder text contrast against form field baggrund
- Danger color border on brand colors

## Describe the solution you'd like
Label buttom padding has been adjusted to 4px
The md field corner radius has been updated to 12px (this is a new size in the corner radius tokens)
A high-contrast stroke has been added to the field frame
Field color have been updated across all states
Focus is set to "Focus inner shadow md"
Alert styling has been revised to align with the focus treatment, ensuring high contrast across all brands
Compined alert and focus states have been updated 

## Are there any additional context?
https://www.figma.com/design/bTihgLkrfmI3AkiaeEuUCF/Accessibility-design-assessment?m=auto&node-id=2252-1764&t=x15saRNke1RDQR7a-1

<hr />

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#the-process-of-contributing) correctly.

### Refinement

- [ ] Request that the issue is [UX refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#ux-refinement)
- [ ] Request that the issue is [tech refined](https://github.com/kirbydesign/designsystem/blob/develop/.github/CONTRIBUTING.md#tech-refinement); do not proceed until this is done.

### Implementation 
The contributor who wants to implement this issue should: 

- [ ] Make sure you have read: "[Before you get coding](../CONTRIBUTING.md/#before-you-get-coding)".
- [ ] Signal to others you are working on the issue by assigning yourself.
- [ ] Create a branch from the [develop branch](https://github.com/kirbydesign/designsystem/tree/develop) following our [branch naming convention](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Branch). 
- [ ] Publish a WIP implementation to Github as a draft PR and ask for feedback. 
- [ ] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [ ] Update the [cookbook](https://cookbook.kirby.design) with examples and showcases.

### Review
Once the issue has been implemented and is ready for review:

- [ ] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review). 
- [ ] Create a pull-request. If you created a draft PR during implementation you can just mark that as "ready for review".

**Comment by henrikvoetmand** [comment]
- Label buttom padding has been adjusted to 4px
- A high-contrast stroke has been added to the field frame
- Field that is active has input-color stroke with no focus.
- Focus is only visible by keyboard
- Focus is set to "Focus inner shadow md"
- Active state is updated
- Placeholder color in theme-colors is updated
- Text-placeholder color is set to kirby-input-color on hover.

**Figma file:**
https://www.figma.com/design/6Fgk0gzWGxJYPYNS0RdDgX/branch/0da7xRA5VWf1quC1oA5pPh/01-Kirby-Components?m=auto&node-id=371-845&t=ESz4j3O6mIObEn1t-1

**Comment by RasmusKjeldgaard** [comment]
Closing this issue automatically because it has been open for more than 6 months and is not currently in progress. If this is still relevant, please feel free to reopen it or open a new issue with updated details. Thank you!
